php/commands/secure.php

```diff
@@ -36,11 +36,59 @@ public function secure_port(){
 		//***This function Secures port***
 		//todo:
 
+		EE::info("Please Enter valid port number");
+		$port = EE::input_value("EasyEngine admin port [22222]:");
+		if(empty($port)){
+			EE::debug("No port selected using default port : 22222");
+			$port = 22222;
+		}
+		
+		while(!is_numeric($port)){
+			EE::debug("Not a valid port");
+			EE::error("Please enter valid port number:");
+			$port = EE::input_value("EasyEngine admin port [22222]:");
+		}
+		EE::debug("Checking distro of the machine");
+		$distro_name = EE_OS::ee_platform_distro();
+		EE::debug("$distro_name found");
+		if("ubuntu" == $distro_name) {
```

Comment:
The _ubuntu_ and _debian_ distro seem to have same settings for this config. Can be put in one **if** block.